projects/prosoft-components-demo/tsconfig.app.json

```diff
@@ -1,13 +1,9 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "outDir": "../../out-tsc/app",
     "types": []
   },
-  "include": ["src/**/*.ts"],
-  "exclude": ["src/test.ts", "src/**/*.spec.ts"],
```

Comment:
Not so sure whether this should be removed or not.

Reply:
The angular migration removed this